src/config.rs

```diff
@@ -711,6 +711,14 @@ impl Context {
         if key == Config::SentboxWatch {
             self.last_full_folder_scan.lock().await.take();
         }
+
+        if matches!(
```

Comment:
Should be moved to `set_config_ex()`, tests on config synchronisation should catch this